src/improvements/template/OPCMUpgradeV200.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {OPCMBaseTask} from "../tasks/OPCMBaseTask.sol";
+import {SuperchainAddressRegistry} from "src/improvements/SuperchainAddressRegistry.sol";
+import {
+    IOPContractsManager,
+    ISystemConfig,
+    IProxyAdmin
+} from "@eth-optimism-bedrock/interfaces/L1/IOPContractsManager.sol";
+import {IStandardValidatorV200} from "@eth-optimism-bedrock/interfaces/L1/IStandardValidator.sol";
+import {Claim} from "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {VmSafe} from "forge-std/Vm.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+
+/// @notice This template supports OPCMV200 upgrade tasks.
+contract OPCMUpgradeV200 is OPCMBaseTask {
+    using stdToml for string;
+    /// @notice The OPContractsManager address
+
+    address public OPCM;
+
+    /// @notice The StandardValidatorV200 address
+    IStandardValidatorV200 public STANDARD_VALIDATOR_V200;
+
+    /// @notice Struct to store inputs for OPCM.upgrade() function per l2 chain
+    struct OPCMUpgrade {
+        Claim absolutePrestate;
+        uint256 chainId;
+    }
+
+    /// @notice Mapping of l2 chain IDs to their respective prestates
+    mapping(uint256 => Claim) public opcmUpgrades;
```

Comment:
```suggestion
    mapping(uint256 => Claim) public absolutePrestates;
```

Reply:
See my other comment below. It's just confusing the relationship between absolute prestates and upgrades

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/664/commits/6422e2b4dd35a420b320bd9458a5f5effe2b60c8